stripe/src/test/java/com/stripe/android/cards/BinTest.kt

```diff
@@ -0,0 +1,30 @@
+package com.stripe.android.cards
+
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.CardNumberFixtures
+import kotlin.test.Test
+
+class BinTest {
+    @Test
+    fun `create() should return expected value`() {
```

Comment:
I think we should split these assertions out into [individual tests](http://wiki.c2.com/?ArrangeActAssert).